SentimentAnalysis.ipynb

```diff
@@ -0,0 +1,280 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sentiment Classification with ThirdAI's Universal Deep Transformer\n",
+    "\n",
+    "To run this notebook, you will need to obtain a ThirdAI license at the following link if you have not already: https://www.thirdai.com/try-bolt/"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip3 install thirdai\n",
+    "!pip3 install datasets\n",
+    "!pip3 install pandas"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dataset Download\n",
+    "\n",
+    "In the next cell, we will process an Amazon "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from datasets import load_dataset\n",
+    "from utils import to_batch\n",
+    "\n",
+    "def load_data(output_filename, split, return_inference_batch=False):\n",
+    "    data = load_dataset('amazon_polarity')\n",
+    "    \n",
+    "    df = pd.DataFrame(data[split])\n",
+    "    df = df[['title', 'label']]    \n",
+    "    df.to_csv(output_filename, index=False, sep='\\t')\n",
+    "    \n",
+    "    if return_inference_batch:\n",
+    "        inference_batch = to_batch(df[[\"title\"]].sample(frac=1).iloc[:5])\n",
+    "        return inference_batch\n",
```

Comment:
Can we move this to utils?

Reply:
I did not want to put in utils for now because we introduce a dependency on the `datasets` library which will break the other demos unless we install `datasets` in every notebook. I was thinking we could do another change where we install a common `requirements.txt` file across all notebooks.